spec/components/after_sign_off_component_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe AfterSignOffComponent, type: :component do
+  let(:assessor) { create(:user, :assessor, name: "John") }
+
+  it "renders nothing when no recommendation" do
+    pa = create(:planning_application)
+
+    render_inline(described_class.new(planning_application: pa))
+
+    expect(page).not_to have_selector("body")
```

Comment:
[How do you test if there's no component being rendered? I didn't know how to do that in RSpec until some random dude answered it on stackoverflow. The answer was superlative but I can only vote once. if you have account please upvote the answer!](https://stackoverflow.com/a/74637090/1511504)